website/test-balance/src/domain/gear.service.ts

```diff
@@ -34,36 +33,64 @@ export class GearService {
         },
       );
     });
-    log.info(`Connected to ${await this.api.chain()} with genesis ${this.genesisHash}`);
-    this.account = await GearKeyring.fromSeed(config.gear.accountSeed);
-    this.rootAccount =
-      config.gear.rootAccountSeed === '//Alice'
-        ? await GearKeyring.fromSuri('//Alice')
-        : await GearKeyring.fromSeed(config.gear.rootAccountSeed);
+    log.info(`Connected to ${await this.api.chain()} with genesis ${this.getGenesisHash}`);
+
+    const [account, rootAccountByGearKeyring] = await Promise.all([
+      GearKeyring.fromSeed(config.gear.accountSeed),
+      this.setRootAccountSeedByGearKeyring(),
+    ]);
+
+    this.account = account;
+    this.rootAccount = rootAccountByGearKeyring;
     this.accountBalance = new BN(config.gear.accountBalance);
     this.balanceToTransfer = new BN(config.gear.balanceToTransfer);
-    if (await this.accountBalanceIsSmall()) {
-      this.transferBalance(this.account.address, this.rootAccount, this.accountBalance);
+    if (await this.isSmallAccountBalance()) {
+      await this.transferBalance(this.account.address, this.rootAccount, this.accountBalance);
+    }
+  }
+
+  public async transferBalance(to: string, from: KeyringPair, balance = this.accountBalance) {
+    await this.api.balance.transfer(to, balance);
+    await this.transferData(from).catch((err) => log.error(err));
+
+    if (to !== this.account.address) {
+      await this.dbService.setTransferDate(to, this.getGenesisHash);
     }
+    return { status: 'ok', transferredBalance: balance.toString() };
   }
 
-  get genesisHash(): string {
+  public get getGenesisHash(): string {
     return this.api.genesisHash.toHex();
   }
 
-  async accountBalanceIsSmall(): Promise<boolean> {
+  private async transferData(from: KeyringPair): Promise<TransferData> {
```

Comment:
let's call it sendTransaction or smth like this. If we make it as a separate method